src/mongoose.ts

```diff
@@ -5,8 +5,10 @@ import logger from './logger';
 export default function (app: Application): void {
   mongoose.connect(
     app.get('mongodb'),
-    { useCreateIndex: true, useNewUrlParser: true }
-  ).catch(err => {
+    { useCreateIndex: true, useNewUrlParser: true, useUnifiedTopology: true }
```

Comment:
can you explain what the `useUnifiedTopology: true` option does? Not saying we shouldn't have it, just trying to figure out the reason for adding it 😃

Reply:
Sorry I left the answer to this in the general comments. I didn't see it here.

Reply:
ahh got it, the current server discovery and monitoring engine is going to be deprecated in version 4.x of MongoDB.
The release notes helped to understand the why: https://github.com/mongodb/node-mongodb-native/releases/tag/v3.2.1 😃

Thank you for explaining @mstanciu552, we should definitely have it 👍

Reply:
Thanks for letting me know.
👍